.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: java
-env:
-  - MYSQL_VERSION=5.5
-  - MYSQL_VERSION=5.6
+branches:
+  only: master
```

Comment:
to be more clear, gets rid of the /push build:
![screen shot 2016-09-07 at 11 22 10 am](https://cloud.githubusercontent.com/assets/697824/18323658/6f66ad00-74ed-11e6-9454-80801a680c0a.png)